teach_base/lesson_02/05_day_of_the_year.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Three numbers are specified, which indicate the day, month, year
+# (we ask the user). Find the ordinal date starting from the beginning
+# of the year. Take into account that the year may be a leap year.
+# A leap year, if it is divisible by four, but if it is divided by
+# 100 without remainder, it is not a leap year. However, if it is divisible
+# by 400, it is a leap year.(It is forbidden to use built-in ruby methods
+# for this like Date # yday or Date # leap?)
+
+puts 'Please input the date(day, month, year)!'
+day = gets.chomp.to_i
+month = gets.chomp.to_i
+year = gets.chomp.to_i
+months = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
+
+def leap_feb(year, months)
+  months[1] = 29 if ((year % 4).zero? && year % 100 != 0) || (year % 400).zero?
+end
+
+leap_feb(year, months)
+
+def correct_date?(day, month, months)
```

Comment:
I expect `correct_date?` to return `true` if the date is correct.
Either rename it to `incorrect_date?` or change the condition.